docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   postgres:
-    image: postgres:9.3
+    image: postgres
```

Comment:
@s-kennedy This should be 9.3, or at least until we can upgrade to a later version. If you're having trouble with the db, do a `docker-compose rm -v postgres` and then recreate the database and import the data. You can import any old db dump, but you'll need to run migrations again.